app/controllers/challenge/gift_exchange_controller.rb

```diff
@@ -64,10 +64,61 @@ def destroy
+      :requests_num_allowed, :offers_num_required, :offers_num_allowed,
+      :signup_instructions_general, :signup_instructions_requests,
+      :signup_instructions_offers, :request_url_label, :offer_url_label,
+      :offer_description_label, :request_description_label, :works_reveal_at_string,
+      :authors_reveal_at_string,
+      request_restriction_attributes: [
+        :id, :optional_tags_allowed, :title_required, :title_allowed, :description_required,
+        :description_allowed, :url_required, :url_allowed, :fandom_num_required,
+        :fandom_num_allowed, :allow_any_fandom, :require_unique_fandom,
+        :character_num_required, :character_num_allowed, :allow_any_character,
+        :require_unique_character, :relationship_num_required, :relationship_num_allowed,
+        :allow_any_relationship, :require_unique_relationship, :rating_num_required,
+        :rating_num_allowed, :allow_any_rating, :require_unique_rating,
+        :category_num_required, :category_num_allowed, :allow_any_category,
+        :require_unique_category, :freeform_num_required, :freeform_num_allowed,
+        :allow_any_freeform, :require_unique_freeform, :warning_num_required,
+        :warning_num_allowed, :allow_any_warning, :require_unique_warning
+      ],
+      offer_restriction_attributes: [
+        :id, :optional_tags_allowed, :title_required, :title_allowed,
+        :description_required, :description_allowed, :url_required, :url_allowed,
+        :fandom_num_required, :fandom_num_allowed, :allow_any_fandom,
+        :require_unique_fandom, :character_num_required, :character_num_allowed,
+        :allow_any_character, :require_unique_character, :relationship_num_required,
+        :relationship_num_allowed, :allow_any_relationship, :require_unique_relationship,
+        :rating_num_required, :rating_num_allowed, :rating_num_required, :allow_any_rating, :require_unique_rating,
+        :category_num_required, :category_num_allowed, :allow_any_category,
+        :require_unique_category, :freeform_num_required, :freeform_num_allowed,
+        :allow_any_freeform, :require_unique_freeform, :warning_num_required,
+        :warning_num_allowed, :allow_any_warning, :require_unique_warning,
+        :tag_sets_to_add, :character_restrict_to_fandom,
+        :character_restrict_to_tag_set, :relationship_restrict_to_fandom,
+        :relationship_restrict_to_tag_set,
+        tag_sets_to_remove: []
+      ],
+      potential_match_settings_attributes: [
+        :id, :num_required_prompts, :num_required_fandoms, :num_required_characters,
+        :num_required_relationships, :num_required_freeforms, :num_required_categories,
+        :num_required_ratings, :num_required_warnings, :include_optional_fandoms,
+        :include_optional_characters, :include_optional_relationships,
+        :include_optional_freeforms, :include_optional_categories, :include_optional_ratings,
+        :include_optional_warnings, :max_tags_matched
```

Comment:
I agree that max_tags_matched is not needed. @davidstump well done in doing that. It took a long time to verify that. Thank you

I am happy to approve with the change sarken has requested.